admin/hotlist-service/pom.xml

```diff
@@ -289,8 +289,8 @@
 							</execution>
 						</executions>
 						<configuration>
-							<apiDocsUrl>http://localhost:8090/app/generic/v3/api-docs</apiDocsUrl>
-							<outputFileName>openapi.json</outputFileName>
+							<apiDocsUrl>http://localhost:8090/app/generic/v3/api-docs/HOTLIST</apiDocsUrl>
+							<outputFileName>${artifactId}-openapi.json</outputFileName>
```

Comment:
why artifactId is used instead of name here ?

Reply:
Yes, actually service name contains space so in  json file name space will come. and as told by mandeep he wrote script for generating html pages. so shell takes space as another input. so i used artifact-id as it doesn't have space.